Examples/ClientTest/ClientTest.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.0;netcoreapp2.0;net47;net46;net45;net4;net35</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0;netcoreapp2.0;net47;net46;net45;net4</TargetFrameworks>
```

Comment:
Can you please reinstate all the net35 targets, then the CI process will succeed

Reply:
done